packages/terra-core-docs/src/terra-dev-site/doc/tag/About.1.doc.mdx

```diff
@@ -42,9 +43,17 @@ import Tag from 'terra-tag';
 * [Mobile Support](https://engineering.cerner.com/terra-ui/about/terra-ui/component-standards#mobile-support)
 
 ## Examples
-<TagDefault title="Default Tag with onClick" />
-<TagHref title="Tag with href" />
-<TagFallbacks title="Tag with no onClick or href" description="These styles are provided for when this component is missing recommended elements for best practice use." />
+
+### Default Tag with associated action
+Tag in it's default form is interactive component designed to provide additional information or trigger actions when clicked. Each tag visually communicates its purpose through text and optional icons.
+<TagDefault title="Default clickable Tag" description="This tag displays a simple text label, and when clicked, it triggers a notification to provide aditional information." />
+<TagIcon title="Default Tag with Icon" description="This tag combines an icon and text label to convey additional meaning. When clicked, it also triggers a notification to inform the user about the interaction." />
+
+### Tag with clickable link
+<TagHref title="Tag with href" description="This tag provides a direct link to a specific webpage. Just click on the tag, and it will seamlessly redirect you to the associated website." />
```

Comment:
```suggestion
<TagHref title="Tag with href" description="This tag provides a direct link to a specific webpage. Just click on the tag, and it will seamlessly redirect to the associated website." />
```